### PR TITLE
Added submission requirements BACK to what they used to be

### PR DESCRIPTION
**Pull request by marlomajor**

### CODE REVIEW

File: 1508/module_4_assignments/ideabox2.0/README.md [hunk 1]
```diff
@@ -1,14 +1,3 @@
-# Ideabox 2.0 Submission Form
-[Project Spec](https://github.com/turingschool/curriculum/blob/master/source/projects/revenge_of_idea_box.markdown)
-
```
Comment (rrgayhart) [inline 1508/module_4_assignments/ideabox2.0/README.md]: It looks like the README deletion is still in this PR!

**Comment by marlomajor** [comment]
I promise I'm just tired. I know how to Git. Well...not really.

**Comment by marlomajor** [comment]
I promise I'm not useless.

**Comment by rrgayhart** [comment]
Bahaha - it's good now, go to sleep!

**Pull request merged**